docs/BR.md

```diff
@@ -355,16 +356,18 @@ The binding SHALL use a digital signature algorithm or a cryptographic hash algo
 
 **Subscriber**: A natural person or Legal Entity to whom a Certificate is issued and who is legally bound by a Subscriber Agreement or Terms of Use.
 
-**Subscriber Agreement**: An agreement between the CA and the Applicant/Subscriber that specifies the rights and responsibilities of the parties.
+**Subscriber Agreement**: An agreement between the TSP and the Applicant/Subscriber that specifies the rights and responsibilities of the parties.[Comment: The TSP is the organization with whom a Subscriber Agreement is entered.]
 
 **Subsidiary Company**: A company that is controlled by a Parent Company.
 
 **Technically Constrained Subordinate CA Certificate**: A Subordinate CA certificate which uses a combination of Extended Key Usage settings and Name Constraint settings to limit the scope within which the Subordinate CA Certificate may issue Subscriber or additional Subordinate CA Certificates.
 
-**Terms of Use**: Provisions regarding the safekeeping and acceptable uses of a Certificate issued in accordance with these Requirements when the Applicant/Subscriber is an Affiliate of the CA or is the CA.
+**Terms of Use**: Provisions regarding the safekeeping and acceptable uses of a Certificate issued in accordance with these Requirements when the Applicant/Subscriber is an Affiliate of the TSP or is the TSP.[Comment: Similar to the Subscriber Agreement, Terms of Use are entered with a TSP.]
 
 **Test Certificate**: A Certificate with a maximum validity period of 30 days and which: (i) includes a critical extension with the specified Test Certificate CABF OID  (2.23.140.2.1), or (ii) is issued under a CA where there are no certificate paths/chains to a root certificate subject to these Requirements.
 
+**Trust Service Provider**: An organization providing trust services, through a number of Certification Authorities, to its customers who may act as Subscribers or Relying Parties.  [Comment:  Do we want to  include Relying Parties?]
```

Comment:
I would make a reference here to http://www.etsi.org/deliver/etsi_en/319400_319499/31941101/01.02.00_20/en_31941101v010200a.pdf chapter 5.4.1 that very clearly defines the term TSP

Reply:
So there was a long discussion about this.  The CA/B Forum definition and usage of TSP is not the ETSI usage of TSP.  They are similar, but different.  I actually argued that it will be confusing to have the same term used in two different but closely related documents, but I lost that fight.  It turns out ISO also uses TSP, with its own definition.

Reply:
Exactly. The TSP definition is also included in ISO 21188 which is referenced as a core document in WebTrust for CAs 2.0. So, this seems to be the common ground between the two audit standards (WebTrust and ETSI), at least for the interpretation of the Baseline Requirements.

Reply:
@timfromdigicert Do you have a reference where exactly ISO uses TSP in a different definition as ETSI? Maybe one could get in contact with ETSI and ask them to align their definition with the definition of ISO. The situation reminds me a little bit on this: https://xkcd.com/927/

Reply:
Ok, so this is why using TSP was a very, very bad idea.  The purpose of this project was *NOT* and was *NEVER* intended to be about aligning the BRs with ISO or ETSI terminology.  That's a much bigger and longer project.

The purpose of this project was to separate out two existing concepts in the BRs, that both used the term CA: the organizational unit that performs CA operations and issues certificates, and the legal entity responsible for that organizational unit.

If it's going to continue to confuse people, we're going to have to go back to "CA operator" instead of TSP.

Reply:
I just finished the preparation of my annual ETSI audit and I would have been highly confused, if there would have been one three-letter abbreviation ('TSA') with a (slightly) different meaning in two documents (ETSI EN 319 411-1 / ETSI EN 319 401 and BRG), which are closely coupled.

Reply:
@timfromdigicert, TSP *is* the legal entity responsible for the operations of CAs (which are organizational units that manage specific CA Keys, have specific practices, etc). I am very happy with this definition and I think it makes things clearer than the "CA Operator" (which contains the term "CA" :)